libs/index/odc/index/stac.py

```diff
@@ -86,9 +150,7 @@ def stac_transform(input_stac):
     """ Takes in a raw STAC 1.0 dictionary and returns an ODC dictionary
     """
 
-    product_label, product_name, region_code = _stac_product_lookup(input_stac)
-
-    deterministic_uuid = str(odc_uuid("sentinel-2_stac_process", "1.0.0", [product_label]))
+    product_id, product_label, product_name, region_code = _stac_product_lookup(input_stac)
```

Comment:
Looks like `product_id` contains UUID of a dataset, it's not a good name for that.

Reply:
Removed ambiguous use of `product_id`